susemanager/src/mgr-salt-ssh

```diff
@@ -3,6 +3,20 @@
 Execute the salt ssh system
 """
 
+# Copyright 2025 SUSE LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
We could use the same format we use in other files, that is only add the [spdx header](https://spdx.dev/learn/handling-license-info/). See https://github.com/uyuni-project/uyuni/blob/0b1b4434231a7b5036f809c74065e186afd31ee2/rel-eng/chart_push.sh#L3-L5 for an example.